### ISSUE TITLE
fix: Latest version of Bun seems to break Devtools

### ISSUE THREAD

**Issue by arapl3y** [nuxt/devtools#494]
### 🐛 The bug

I've spun up a fresh Nuxt 3 project using Bun according to the docs and the devtools appear to be broken.

I get a 500 response in the UI 
<img width="1293" alt="Screenshot 2023-10-26 at 5 02 26 pm" src="https://github.com/nuxt/devtools/assets/14939268/16842ffe-9dcf-409b-8c1d-614e0ae89d0f">

and in the console I get the following error

```
 ERROR  [h3] [unhandled] parse is not a function

  at node_modules/sirv/build.js:168:18
  at call (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50689:7)
  at next (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50633:5)
  at next (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50611:14)
  at call (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50698:3)
  at next (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50633:5)
  at viteServeStaticMiddleware (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:47674:20)
  at viteServeStaticMiddleware (node_modules/vite-plugin-inspect/dist/index.mjs:834:24)
  at call (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50689:7)
  at next (node_modules/vite/dist/node/chunks/dep-bb8a8339.js:50633:5)
```

### 🛠️ To reproduce

Stackblitz doesn't support Bun

### 🌈 Expected behavior

Devtools work the same way they work with npm or yarn.

### ℹ️ Additional context

Nuxt 3.8.0
Bun 1.0.7
Node 20.9.0 and 18.17.0
M2 MacBook Pro
Ventura 13.5
Google Chrome Version 118.0.5993.117 (Official Build) (arm64)

Confirmed yarn, npm and pnpm work well.

**Comment by arashsheyda** [comment]
what's your Nuxt version? I'm using the latest version of Nuxt and DevTools and it's working well.

**Comment by arapl3y** [comment]
> what's your Nuxt version? I'm using the latest version of Nuxt and DevTools and it's working well.

3.8.0

**Comment by makenowjust** [comment]
**Quick solution**: add the following to your `package.json` and run `bun install`.

```json
  "overrides": {
    "@polka/url": "1.0.0-next.23"
  }
```

- - -

I faced the same error and found the reason for this.

Currently, `bun` installs `@polka/url@1.0.0-next.9` as a dependency of `sirv`. However, `sirv` requires `@polka/url@^1.0.0-next.20` and they are different signatures so that it causes the error. `bun`'s version resolution for prerelease tags seems buggy ([related issue?](https://github.com/oven-sh/bun/issues/3684)).

There are two solutions:

1. wait for `bun` to be fixed,
2. fix `@polka/url` version using `overrides` or `resolutions` ([doc](https://bun.sh/docs/install/overrides)).

So, I suggested the second in the above of this comment. Thank you.
